leaderboard/tie_ranking_leaderboard.py

```diff
@@ -300,3 +286,44 @@ def _ties_leaderboard_key(self, leaderboard_name):
         @return a key in the form of +leaderboard_name:ties_namespace+
         '''
         return '%s:%s' % (leaderboard_name, self.ties_namespace)
+
+    def _members_from_rank_range_internal(
+            self, leaderboard_name, start_rank, end_rank, members_only=False, **options):
+        '''
+        Format ordered members with score as efficiently as possible.
+        '''
+        response = self._range_method(
+            self.redis_connection,
+            leaderboard_name,
+            start_rank,
+            end_rank,
+            withscores=not members_only)
+
+        if members_only or not response:
+            return [{self.MEMBER_KEY: member} for member in response]
+
+        current_member, current_score = response[0]
```

Comment:
I think more self-documenting variable names here might be: `starting_member`, `starting_score`, and `starting_rank`.